src/transformers/fiddle-embedder.js

```diff
@@ -84,6 +99,10 @@ function getFiddleAST(dir, version) {
     return children;
   }
 
+  if (!fileNames.includes(focus)) {
+    throw new Error(`Provided focus (${focus}) is not an available file in this fiddle (${dir}). Available files are [${fileNames.join(",")}]`);
```

Comment:
nit: slightly more human-readable

```suggestion
    throw new Error(`Provided focus (${focus}) is not an available file in this fiddle (${dir}). Available files are [${fileNames.join(", ")}]`);
```